avm/ptn/authorization/pim-role-assignment/modules/management-group.bicep

```diff
@@ -37,7 +37,7 @@ param targetRoleAssignmentScheduleInstanceId string = ''
 param ticketInfo ticketInfoType?
 
 @sys.description('Optional. The conditions on the role assignment. This limits the resources it can be assigned to.')
-param condition string?
+param condition string = ''
```

Comment:
Are you sure this is required? Mind you `empty(null)` is results in false, just as `empty('')`. So I think you can leave the `condition` as is and just use the conditionVersion check.

This would, by the way, also match the way the `roleAssignments` module handels the two values:
https://github.com/Azure/bicep-registry-modules/blob/7da75993e1864623b115a2a954f9d567279f5835/avm/res/analysis-services/server/main.bicep#L151-L152

Reply:
Sorry for chiming in here @ReneHezser, just following up on @sebassem's earlier [PR](https://github.com/Azure/bicep-registry-modules/pull/4431#issuecomment-2677894010) for this module that we merged yesterday :)

Reply:
@AlexanderSehr makes sense, just updated this parameter